### PR TITLE
Added type hints to additional tests

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

**Comment by hugovk** [comment]
The CI is failing for Ubuntu with all versions (except PyPy). For example:

```python
ERROR Tests/test_image_fromqimage.py::test_sanity_1 - Failed: fixture function has more than one 'yield':

    @pytest.fixture
    def test_images() -> Generator[Image.Image, None, None]:
        ims = [
            hopper(),
            Image.open("Tests/images/transparent.png"),
            Image.open("Tests/images/7x13.png"),
        ]
        try:
            yield from ims
        finally:
            for im in ims:
                im.close()
/home/runner/work/Pillow/Pillow/Tests/test_image_fromqimage.py:21
FAILED Tests/test_image_fromqimage.py::test_sanity_1 - TypeError: 'PpmImageFile' object is not iterable
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 2 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by radarhere** [comment]
Ok, I've updated the commit so that it passes now.

**Review by hugovk — APPROVED** [review]
Thanks!

File: Tests/test_file_spider.py [hunk 1]
```diff
@@ -152,12 +152,13 @@ def test_nonstack_dos() -> None:
             assert i <= 1, "Non-stack DOS file test failed"
 
 
-# for issue #4093
+# for issue #4093s
```
Comment (nulano) [inline Tests/test_file_spider.py:155]: typo?

Comment (radarhere) [reply Tests/test_file_spider.py:155]: Oops, thanks. I've created #7855

**Pull request merged**